library/tartarus_lib/clients/config_definition.py

```
@@ -0,0 +1,105 @@
+    """
+
+    def __init__(self, base_client: BaseAPIClient):
+        """
+        Initialize the ConfigDefinitionAPI.
+
+        :param base_client: An instance of BaseAPIClient.
+        """
+        self.base_client = base_client
+
+    def create(
+        self, config_definition_key: str, json_schema: dict = None, indexes: list = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)